src/pfe/portal/routes/projects/remoteBind.route.js

```diff
@@ -320,7 +329,6 @@ async function listFiles(absolutePath, relativePath) {
     const nextRelativePath = path.join(relativePath, f);
```

Comment:
perhaps out of scope but fyi there are open source node modules that do this function (better than us) e.g. https://www.npmjs.com/package/recursive-readdir-async